api/src/services/platform-service.ts

```diff
@@ -633,6 +635,80 @@ export class PlatformService extends DBService {
     };
   }
 
+  /**
+   * Uploads the given survey occurrence input file to BioHub
+   *
+   * @param {number} surveyId
+   * @param {string} dataPackageId
+   * @memberof PlatformService
+   */
+  async submitSurveyObservationInputDataToBiohub(surveyId: number, dataPackageId: string) {
+    const surveyService = new SurveyService(this.connection);
+    const occurrenceSubmissionData = await surveyService.getLatestSurveyOccurrenceSubmission(surveyId);
+
+    if (!occurrenceSubmissionData?.output_key) {
+      throw new ApiGeneralError('Failed to submit survey to BioHub', ['Occurrence record has invalid s3 output key']);
+    }
+    const s3File = await getFileFromS3(occurrenceSubmissionData.output_key);
```

Comment:
This is being called here, but `s3File` is never used. In the `_makeArtifactFromObservationInputData` function we make this call again.